package.json

```diff
@@ -13,6 +13,14 @@
   "keywords": [],
   "author": "Rebecca Turner <me@re-becca.org> (http://re-becca.org/)",
   "devDependencies": {},
+  "peerDependencies": {
+    "bluebird": "^2.0.0 || ^3.0.0"
```

Comment:
Last v1 release was 6 years ago, so I think it's safe to start from v2. We can do `*` or `>=`, but I think it's better to specify explicit range.